### ISSUE TITLE
YOLOv11-n model false detection

### ISSUE THREAD

**Issue by WYL-Projects** [ultralytics/ultralytics#19118]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hello author, 
I trained a pedestrian object detection model using my own annotated pedestrian dataset, based on the YOLOV11-n model. The training commands are as follows:
`yolo detect train data=mydata.yaml model=weights/yolov11_det/yolo11n.pt batch=64 device=0,1 imgsz=640 epochs=100 project=runs/detect/pedestrian_detection_yolov11 name=v0 workers=16 degree=45 save_period=10`

The content of vstd.yaml is as follows:

> path: /data/object_detection/pedestrian_detection
> train: train_v2.txt
> val: val.txt
> test: 
> 
> names:
> 0 person

When I use the trained best. pt model to infer images, the model predicts the background of some small targets (such as distant wall corners) as Person class, as shown in the figure below, with a confidence level exceeding 0.5. I try to filter out false targets with high confidence when reasoning, but it may result in some positive samples being filtered out. Is there a better solution to this problem?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @WYL-Projects, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it. This includes details about your dataset, the training process, and any specific observations about the issue during training or inference.

For your specific question regarding false detections by the YOLOv11-n model, here are some suggestions: 
- Share example input images with and without the issue so we can further understand your case.
- Confirm you have followed the [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to ensure optimal training performance. This includes considerations around data quality, augmentation strategies, and hyperparameter tuning.
- You may also want to visually inspect your dataset annotations to ensure accurate label quality as even small inconsistencies can lead to unintended predictions.

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://reddit.com/r/Ultralytics) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

☝️ This is an automated response, but an Ultralytics engineer will assist you further soon. Thank you for your patience and for helping us improve! 😊

**Comment by Y-T-G** [comment]
Your dataset is probably small and not diverse, so the model overfit. You need to improve the dataset and add background images.

https://docs.ultralytics.com/yolov5/tutorials/tips_for_best_training_results/

**Comment by WYL-Projects** [comment]
@Y-T-G Thanks for the reply and advice, I'll give it a try

**Comment by Y-T-G** [comment]
@WYL-Projects Is that with augmentations or without?

**Comment by WYL-Projects** [comment]
@Y-T-G The default data enhancements in default.yaml are used. The default data enhancements in default.yaml are used. I found an image of the misdetection as shown below:

![Image](https://github.com/user-attachments/assets/42a5c967-26af-43e5-a3f1-b3a4ba7fcb40)

**Comment by Y-T-G** [comment]
Does your dataset contain fisheye images?

**Comment by WYL-Projects** [comment]
yes, it's worth mentioning that half of the samples I trained on were fisheye images and I added the degrees=45 parameter to randomly rotate the images

**Comment by Y-T-G** [comment]
Maybe you should remove random rotation because it's bad for object detection

**Comment by Y-T-G** [comment]
And you should train a larger model like yolo11m

n is very small

**Comment by WYL-Projects** [comment]
@Y-T-G Ok, thanks for the patient reply, I'll try that later. So I'm curious what kind of scenarios this random rotation is usually used under? I started using random rotation for data enhancement because I was doing pedestrian detection on the original fisheye image, and the pedestrians were a bit skewed looking with large distortions.

**Comment by Y-T-G** [comment]
It's useful if you have a segmentation dataset. For bounding boxes, the box will be wrong after rotation unless you perform 90 degree rotations.

https://github.com/ultralytics/ultralytics/issues/18304

**Comment by WYL-Projects** [comment]
@Y-T-G Thanks for the explanation! I also found the inaccuracy of the target frame after rotation in train_labels.jpg, I will follow up to correct this error. I have another request, I've recently been using yolov11 to go for fisheye target detection, can you provide me with some other suggestions or points to look out for? For example, data enhancement, and improvements in modeling etc. I'll be glad to hear from you.

**Comment by WYL-Projects** [comment]
@Y-T-G I set degrees to 0 and retrained the model, here are my training results:

![Image](https://github.com/user-attachments/assets/32052df3-3962-4038-826b-6d2ac7ca0f2a)

I have a question that the first three epoch val_loss is going up all the time and the precision and recall is going down, is this normal?

**Comment by Y-T-G** [comment]
You can try using `warmup_epochs=0`

**Comment by WYL-Projects** [comment]
@Y-T-G 
How can I implement the following transfer learning:
I have a yolov10-n model A that only detects Person, and now I want to add the categories car, truck, bus, motorcycle, bike, etc. to that model. However, I don't want to reduce the effectiveness of the model in detecting Person, how do I go about training this new model. Looking forward to your suggestions, thanks!

**Comment by Y-T-G** [comment]
This isn't transfer learning. This is continual learning and it's not possible to add classes to a model without affecting the performance of existing classes, unless you add extra parameters for the new classes. But even then, the performance of new classes would depend on whether the features required for those classes were already learnt. If they weren't, then the new classes would perform poorly.

Since you just trained on person classes, it is unlikely your model learnt diverse features.

**Comment by WYL-Projects** [comment]
Okay, thanks for the correction.

**Comment by WYL-Projects** [comment]
> This isn't transfer learning. This is continual learning and it's not possible to add classes to a model without affecting the performance of existing classes, unless you add extra parameters for the new classes. But even then, the performance of new classes would depend on whether the features required for those classes were already learnt. If they weren't, then the new classes would perform poorly.
> 
> Since you just trained on person classes, it is unlikely your model learnt diverse features.

@Y-T-G When you mention here that you add extra parameters for the new classes, you are referring to which parameters to add.

**Comment by Y-T-G** [comment]
Extra parameters like adding an extra layer or extra head for new classes.

https://y-t-g.github.io/tutorials/yolov8n-add-classes/

**Comment by WYL-Projects** [comment]
ok, thanks

**Comment by Y-T-G** [comment]
Freezing will still change performance of person class

**Comment by WYL-Projects** [comment]
@Y-T-G How do I distil yolov11-n with a large model like the generic detection large model GLEE-plus?
